phylogenetic/rules/merge_sequences_usvi.smk

```diff
@@ -43,5 +43,6 @@ rule append_usvi:
           -n accession \
           -e '$genbank_accession' \
         | csvtk concat -tl - {input.usvi_metadata} \
+        | tsv-select -H -f accession --rest last \
```

Comment:
_(non-blocking)_

Suggestion: In cases like this where a column name is ambiguous, add more detail somewhere in the repo. Maybe as the docstring of this rule:

```
rule append_usvi:
    """Appending USVI sequences.

    Notable columns:
    - accession: Either the GenBank accession or USVI accession.
    - genbank_accession: For Auspice to generate a URL to the NCBI GenBank record. Empty for USVI sequences.
    - url: ?
    """
    input:
        …
```

I don't know if this has been done in other repos, but it seems like it'd be useful to bring this context out of commit messages/PRs and into the code itself.

Reply:
Thanks, agree with writing the context into the code itself (docstring). Fixed with https://github.com/nextstrain/zika/pull/36/commits/3631e90e0aeb2847a025f2b4a251ed1a8b7fe075 but let me know if the url explanation is

Reply:
(Discussion is beyond the scope of this PR now, so feel free to merge first.)

The [Auspice source code](https://github.com/nextstrain/auspice/blob/f02583937ea4da8035a0e7a09e5d0e0b420966cc/src/components/tree/infoPanels/click.js#L56) indicates that when both are specified, `genbank_accession` takes precedence and `url` will be ignored.

Because of this behavior, I don't think `url` should be set for GenBank sequences. That would only cause confusion in the event that GenBank changes the URL, we try to update it in this file, and scratch our heads over why the old URL is still showing on Auspice. I would expect `url` to only be set for USVI sequences where there is no `genbank_accession`.